binaries/build.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A python script to automate the Shaka-Streamer-Binaries building
+and packaging process for different platforms."""
+
+import setuptools
+from setuptools.command.build_py import build_py
+import sys
+import shutil
+import os
+import yaml
+
+# os.chmod() is called for every binary at import time,
+# before we put them in wheels.
+import streamer_binaries
+
+
+class custom_build_py(build_py):
+  """A custom class to override the default behavior of `build_py` command."""
+
+  def run(self):
+    # Clean the build directory so the binaries that were added at the previous
+    # build don't remain in the package for the current platform we build for.
+    # This is because setuptools doesn't delete it after a build has
+    # completed and doesn't also re-create it in the next `build` call,
+    # that's why we can't delete it either, so let's just clean it.
+    build_dir = os.path.join(self.build_lib, streamer_binaries.__name__)
+    self._clean_build_dir(build_dir)
+    return super().run()
+
+  def _clean_build_dir(self, build_dir):
+    try:
+      # To clean the directory, we can remove it and re-create it.
+      shutil.rmtree(build_dir)
```

Comment:
If you have time, it would be cool if build.py had constants for the release tags of Packager & ffmpeg, and would download the binaries from the GitHub repos automatically, rather than requiring us to download them manually and in advance of build.py.

Reply:
Sure :)